azurecompute-arm/src/main/java/org/jclouds/azurecomputearm/AzureComputeProviderMetadata.java

```diff
@@ -61,13 +59,11 @@ public static Properties defaultProperties() {
       properties.setProperty(OPERATION_POLL_MAX_PERIOD, "15");
```

Comment:
Seems `AzureManagementApiMetadata.defaultProperties` are getting set 2x, once here and once in `AzureComputeProviderMetadata.defaultProperties()`. Can we keep one set? Is there a reason to have this in two different places?